### ISSUE TITLE
nextCloud News integration possible?

### ISSUE THREAD

**Issue by chesterdkat** [Ranchero-Software/NetNewsWire#2247]
I’d love to see integration with the News app in nextCloud.

**Comment by vincode-io** [comment]
It looks like the project is here: https://github.com/nextcloud/news

**Comment by xplosionmind** [comment]
Yes! I totally second this feature, it’s exactly the one and only thing NetNewsWire lacks, to me. Thanks a lot in advance, looking forward to know more about the development!

**Comment by xplosionmind** [comment]
> It looks like the project is here: https://github.com/nextcloud/news

Yep, but as you can see there’s no "NetNewsWire" in the [Sync Clients](https://github.com/nextcloud/news#sync-clients) list.

**Comment by aniqueta** [comment]
@brentsimmons and team - I'd also love this. Worst case, some self-hosted option. But this is my preference. :)

**Comment by ingorichter** [comment]
I would love to have support for Nextcloud News! That is currently a missing piece and pain-point since I'm reading my feeds on different devices with different rss readers and having one source of truth (Nextcloud News) would be great.

**Comment by timharek** [comment]
Has there any development regarding this issue?

**Comment by brentsimmons** [comment]
@timharek I don’t believe we’ve done any work on this. Right now we’re concentrating on finishing up 6.0 for iOS, rather than adding new features.

**Comment by jomax94** [comment]
> @timharek I don’t believe we’ve done any work on this. Right now we’re concentrating on finishing up 6.0 for iOS, rather than adding new features.

Now that version 6 is released, is there a chance for the Nextcloud News integration?

**Comment by meichthys** [comment]
> 
> 
> @brentsimmons and team - I'd also love this. Worst case, some self-hosted option. What's nice about Nextcloud is the easy install and lightweight system and software requirements. For example, hosting your own Feedbin is quite involved. Doubt it could run on a Raspberry Pi for instance.

If you're looking for a self-hosted option, you could check out [FrechRSS](https://www.freshrss.org/). 
I'd still prefer Nextcloud News though 😉

**Comment by Suzie97** [comment]
Any updates on this?? Please add this feature, I love the app and this is preventing me from using it.

**Comment by Yesfinethankyou** [comment]
Would really love to see Nextcloud News supported as well. Right now I’ve just been running my own FeshRss server just for this but would love to just get everything running in my Nextcloud.

**Comment by meichthys** [comment]
I'm not aware of any work being done on this, but before there is, we may want to wait for the Nextcloud News front-end re-write to be finished. I'm not sure if or how much the re-write may affect the backend, but we may want to wait to see how it plays out:
https://github.com/nextcloud/news/pull/748

**Comment by meichthys** [comment]
Just following up here.. I believe the frontend changes are complete in nextcloud news.

**Comment by xiaolilo** [comment]
Almost a year later, any developments on this? Would really be great.

**Comment by augustebaum** [comment]
Also would really benefit from this feature! Thanks for developing NNW, it's really become an important part of my digital life :)

**Comment by mjeshurun** [comment]
Adding my vote for Nextcloud News support 🙏

**Comment by ostfriese4** [comment]
I need this too for syncing with Linux ([NewsFlash](https://flathub.org/apps/io.gitlab.news_flash.NewsFlash))
